docs/guide/breakpoint-preview.md

```diff
@@ -149,11 +159,14 @@ Advanced features of media queries, including experimental or non-standard ones,
 ### Summary of Unsupported or Partially Supported Features
 | **Media Query Feature**       | **Support in Container Queries**       |
 |-------------------------------|---------------------------------------|
-| `min-height` / `max-height`   | ❌ Unsupported                        |
-| `orientation`                 | ❌ Unsupported                        |
-| `aspect-ratio`                | ❌ Unsupported                        |
-| Complex logical conditions    | ⚠️ Partial / Unreliable               |
-| Viewport-relative units       | ⚠️ Not container-aware               |
-| User preference queries       | ❌ Unsupported (e.g., dark mode)      |
+| `orientation` | ❌ Unsupported |
```

Comment:
All unsupported query feature can be implemented using the `transform` method by the project owner.

`transform` will receive the media query and is supposed to return the new value for breakpoint preview.

In a nutshell, if this is not supported by the modules, you can still implement this yourself using the `transform` option.

Reply:
You make a really good point. I totally forgot about `transform`. That should be documented (or at least mentioned).

Reply:
Section added.